yt_dlp/extractor/dplay.py

```diff
@@ -1062,7 +1063,7 @@ class DiscoveryNetworksDeIE(DiscoveryPlusBaseIE):
             'ext': 'mp4',
             'title': 'German Gold',
             'description': 'md5:f3073306553a8d9b40e6ac4cdbf09fc6',
-            'display_id': 'goldrausch-in-australien/german-gold',
+            'display_id': '4756322',
```

Comment:
As the series/slug is not being passed into `_get_disco_api_info()` for the new case, the `display_id` has either to be re-created there by changing this in the returned dict:
```diff
-            'display_id': display_id,
+            'display_id': '/'.join(traverse_obj(video, (
+                (('included', ...), 'data'), 'attributes', 'alternateId', {strip_or_none}, filter)))
+            if display_id == video_id else display_id,
```
or just re-applied to the returned result (but that would only handle the DMAX case).
 
Then this value should be reverted and the test result for this new test should be have: 
```py
            'display_id': 'feuerwache-3-alarm-in-muenchen/24-stunden-auf-der-feuerwache-3',
```